subprojects/gradle.properties

```diff
@@ -5,11 +5,15 @@ org.gradle.parallel=true
 org.gradle.caching=true
 # https://github.com/gradle/gradle/issues/4823
 org.gradle.configureondemand=false
-org.gradle.unsafe.configuration-cache=true
-# TODO last two errors here: https://youtrack.jetbrains.com/issue/KT-43605
-#  should be ok to use it because properties are `java.rmi.server.hostname` and `kotlin.environment.keepalive`
-#  which should not change on local machine
-org.gradle.unsafe.configuration-cache-problems=fail
+org.gradle.unsafe.configuration-cache=false
+# Kotlin
+# - https://youtrack.jetbrains.com/issue/KT-43605
+#   should be ok to use it because properties are `java.rmi.server.hostname` and `kotlin.environment.keepalive`
+#   which should not change on local machine
+# Detekt
+#  - https://github.com/detekt/detekt/pull/3822
+org.gradle.unsafe.configuration-cache-problems=warn
+org.gradle.unsafe.configuration-cache.max-problems=1
```

Comment:
Is it heuristics or we know the exact problem?

Reply:
Detekt, yes